source/material_model/visco_plastic.cc

```diff
@@ -149,6 +149,11 @@ namespace aspect
                                std::max(std::sqrt(std::fabs(second_invariant(deviator(strain_rate)))),
                                         min_strain_rate) );
 
+      // Choice of activation volume depends on whether there is an adiabatic temperature
+      // gradient used when calculating the viscosity. This allows the same activation volume
+      // to be used in incompressible and compressible models. Default value is 0. Units [K/Pa]
```

Comment:
Please remove the default value and unit in the comment, this has been documented in the function below (and if we ever change this, we will likely forget to update this comment).

Reply:
okay. done.